x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/Warnings.java

```diff
@@ -34,28 +34,39 @@ public void registerException(Exception exception) {
     /**
      * Create a new warnings object based on the given mode
      * @param warningsMode The warnings collection strategy to use
-     * @param source used to indicate where in the query the warning occured
+     * @param source used to indicate where in the query the warning occurred
```

Comment:
I've given us a little more flexibility, specifically so I can tell folks that multivalues in CASE default to `false`. If it's confusing why we do that read the PR description. Er, well, it's still confusing a bit. But that tries to explain it.